eth/gasprice/gasprice.go

```diff
@@ -194,10 +194,12 @@ func (oracle *Oracle) SuggestTipCap(ctx context.Context) (*big.Int, error) {
 	// high-priced txs are causing the suggested tip cap to be high.
 	pendingTxCount, _ := oracle.backend.Stats()
 	if pendingTxCount < oracle.congestedThreshold {
-		// Before Curie (EIP-1559), we need to return the total suggested gas price. After Curie we return 1 wei as the tip cap,
+		// Before Curie (EIP-1559), we need to return the total suggested gas price. After Curie we return 2 wei as the tip cap,
```

Comment:
what about changing the default value to 1 Wei?

Reply:
Can also do that but might have yet again unintended consequences down the line

Reply:
Changed `DefaultIgnorePrice=1` but left the default tip with no congestion at `2 wei` for now. This way, nodes that didn't upgrade will also already consider the lower tip to update the their GPO. Later on we can set the default for no congestion to `1 wei`.